spec/models/cart_spec.rb

```diff
@@ -65,4 +67,20 @@
 
     expect(cart.total_count).to eq(3)
   end
+
+  it ".items in cart returns a collection of item objects" do
+    category = Category.create!(name: "toys")
+    item_1 = category.items.create!(title: "fake wand",
+                                    description: "wow all your friends",
+                                    price: 7)
+    item_2 = category.items.create!(title: "fake love potion",
+                                    description: "wow all your dumb friends",
+                                    price: 7)
+    cart = Cart.new({})
+    cart.add_item(item_1.id)
+    cart.add_item(item_1.id)
+    cart.add_item(item_2.id)
+
+    expect(cart.items).to eq({item_1 => 2, item_2 =>1})
```

Comment:
Do we have fabricator? We should use that for creating categories and items in the tests